posts/first-lean-release.md

```diff
@@ -0,0 +1,49 @@
+---
+author: 'Scott Morrison'
+category: 'announcement'
+date: 2023-08-29 16:30:00 UTC+02:00
+description: 'The first official release of Lean 4, and our plan for regular releases.'
```

Comment:
Is "our" correct here? Is this a lean-community plan or a lean FRO plan?

Reply:
```suggestion
description: 'The first official release of Lean 4, and plans for regular releases.'
```